wordpress.org/public_html/wp-content/plugins/official-wordpress-events/official-events-online.js

```diff
@@ -12,6 +12,7 @@ function getSortedEvents( events ) {
 		// Inject a human-readable start time.
 		event.startTime = wp.date.format( 'g:i a', d );
 		event.timezone = timezone;
+		event.type = event?.type === 'wordcamp' ? 'WordCamp' : toTitleCase(event.type);
```

Comment:
What's the browser support on `?.` ? This file isn't transpiled so we'll need to make sure it meets https://make.wordpress.org/core/handbook/best-practices/browser-support/

Reply:
yeah, I believe it supports the browsers required on the list.

**Filter browsers with a global usage > 1%**
![image](https://github.com/WordPress/wordpress.org/assets/18050944/62985967-33b7-4d98-9928-2128cd53eaf8)


**List created by the rule in [browser support](https://make.wordpress.org/core/handbook/best-practices/browser-support/)**
and_chr 114
android 114
chrome 114
chrome 113
chrome 112
chrome 109
edge 114
edge 113
edge 112
firefox 113
firefox 112
ios_saf 16.5
ios_saf 16.4
ios_saf 16.3
ios_saf 16.1
opera 99
opera 98
safari 16.5
safari 16.4
samsung 20

Reply:
yeah, I believe it supports the browsers required on the list.

**Filter browsers with a global usage > 1%**
![image](https://github.com/WordPress/wordpress.org/assets/18050944/fa2ca33f-8125-4dc8-9654-2665f55f6b82)

**List created by the rule in [browser support](https://make.wordpress.org/core/handbook/best-practices/browser-support/)**
and_chr 114
android 114
chrome 114
chrome 113
chrome 112
chrome 109
edge 114
edge 113
edge 112
firefox 113
firefox 112
ios_saf 16.5
ios_saf 16.4
ios_saf 16.3
ios_saf 16.1
opera 99
opera 98
safari 16.5
safari 16.4
samsung 20

Reply:
👍🏻 thanks for checking!